### ISSUE TITLE
Binary release archives can not be decompressed without error

### ISSUE THREAD

**Issue by jo-chen** [halide/Halide#3670]
Halide binary releases seem to contain duplicate entries: `halide/README_bazel.md`

Decompressing the binary release:
```
cmake -E tar xfz halide-linux-64-gcc53-trunk-46d8e9e0cdae456489f1eddfd6d829956fc3c843.tgz 
CMake Error: Problem with archive_write_header(): Can't create 'halide/README_bazel.md'
CMake Error: Current file: halide/README_bazel.md
CMake Error: Problem extracting tar: halide-linux-64-gcc53-trunk-46d8e9e0cdae456489f1eddfd6d829956fc3c843.tgz
```

This may be a cmake bug, because decompressing using `tar -xzvf` works. However, it would be nice to use cmake ExternalProject_Add to include Halide binary releases in cmake projects. Without the duplicate entries, that would be possible.

**Comment by johnsheu** [comment]
FWIW this has been fixed since 570aad98e39b446d188fdd77072357b6672edb28 on 16 Jan 2019.  The root cause is the Makefile for the binary distribution specifying README_bazel.md twice; GNU tar causes the second reference to be a hardlink to the first (and will decompress it correctly), but other implementation of the tar decompression (including, I assume, the one in CMake, or BSDtar variants on MacOSX) may not.

So it's been fixed, but until these good folks decide to make a new binary release, we're out of luck.

**Comment by steven-johnson** [comment]
Sorry for the inconvenience. We're finally going to be doing another binary release, hopefully today.
